plugin-server/src/utils/db/db.ts

```diff
@@ -1476,6 +1477,21 @@ export class DB {
         return selectResult.rows[0] ?? null
     }
 
+    public async fetchBillingUsage(organizationId: Team['organization_id']): Promise<BillingUsage | null> {
```

Comment:
Wondering if this can be done as part of the team load with a JOIN. Saves an extra API request